readme.md

```diff
@@ -0,0 +1,109 @@
+# GitPress Example
+
+The objective here is to create example project for the ![Gitpress platform](https://github.com/pschoffer/gitpress).
+Its very easy to install and you can try it easily home.
+
+## Instalation
+
+ ### Manual Instalation  	
+ - Unzip the package in an empty directory and upload everything;
+ - Open `wp-admin/install.php` in your browser. It will take you through the process to set up a `wp-config.php` file with your database connection details;
+    - If for some reason this doesn't work, don't worry. It doesn't work on all web browsers. Open up `wp-config-example.php` with a text editor like WordPad or similar and fill in your database connection details;
+    - Save the file as `wp-config.php` and upload it;
+    - Open ![wp-admin/install.php](wp-admin/install.php) in your browser;
+- Once the configuration file is set up, the installer will set up the tables needed for your site. If there is an error, double check your `wp-config.php` file, and try again. If it fails again, please go to the ![WordPress support forums](https://wordpress.org/support/forums/) with as much data as you can gather;  
+- **If you did not enter a password, note the password given to you** If you did not provide a username, it will be `admin`;
+- The installer should then send you to the ![login page](wp-login.php). Sign in with the username and password you chose during the installation. If a password was generated for you, you can then click on `Profile` to change the password.
+
+ ### Using Docker container
+  You can start by fetching the example project.
+
+ ```
+  docker run -p 80:80 -d \
+     --env "GITPRESS_MODE=FETCH" \
+     --env "WORDPRESS_URL=http://localhost/" \
+     --env "GIT_ORIGIN=github.com/pschoffer/gitpress-example.git" \
+     pschoffer/gitpress
+ ```
+
+Docker compose version:
+
+ ```
+ version: '3.1'
+
+ services:
+   gitpress:
+     image: pschoffer/gitpress
+     ports:
+       - 80:80
+     environment:
+       GITPRESS_MODE: FETCH
+       WORDPRESS_URL: http://localhost/
+       GIT_ORIGIN: github.com/pschoffer/gitpress-example.git
+ ```
+
+The admin login credentials are `gitpress:pass`		
+
+## Updating
```

Comment:
Please drop all the sections bellow. Those seems to be related to wordpress itself, but this is only example repo meant to be used as demo for the gitpress itself. 

Although there is some value in comming up with migration flow, it will need to be more complex than that (leveraging the SEED mode of gitpress and then applying changes and persisting them to git).

Reply:
Just tu be clear. 
Drop the whole Updating section and also the "...The admin login credentials..." line?

Reply:
> Just tu be clear.
Drop the whole Updating section and also the "...The admin login credentials..." line?

* Please keep the The admin login credentials are `gitpress:pass`, as that is related to the docker install.
* Drop the update
* Drop Migrating from other systems
* Drop System Requirements (or change to just docker I guess)
* Drop Recommendations
* Drop Online Resources and Final Notes
* Drop Share the Love (or rephrase it to apply to Gitpress instead of WordPress, as the same is true for this project as well:))
* In Licence change Wordpress to either "This Gitpress example" or something along those lines (or drop)